src/linker/Linker/Driver.cs

```diff
@@ -572,6 +573,10 @@ protected int SetupContext (ILogger customLogger = null)
 				return -1;
 			}
 
+			// Default to deterministic output
+			if (!new_mvid_used && !deterministic_used) {
```

Comment:
Does it mean `--deterministic true` will not enable deterministic output

Reply:
No - `--deterministic true` will do exactly the same (it's effectively the default now). See https://github.com/mono/linker/blob/4936cd82cb5f1d438d83c7e06c69ba1ba32eab2e/src/linker/Linker/Driver.cs#L395

What this means is mostly that specifying for example `--new-mvid true` will still take effect even if deterministic is the default (`--new-mvid` effectively overrides the default).

Reply:
Yep, missed that